src/nvim/normal.h

```diff
@@ -10,53 +10,50 @@
 #define FIND_STRING     2       /* find any string (WORD) */
 #define FIND_EVAL       4       /* include "->", "[]" and "." */
 
-/*
- * Arguments for operators.
- */
+// Arguments for operators.
 typedef struct oparg_S {
-  int op_type;                  /* current pending operator type */
-  int regname;                  /* register to use for the operator */
-  int motion_type;              /* type of the current cursor motion */
-  int motion_force;             /* force motion type: 'v', 'V' or CTRL-V */
-  bool use_reg_one;             /* true if delete uses reg 1 even when not
-                                   linewise */
-  bool inclusive;               /* true if char motion is inclusive (only
-                                   valid when motion_type is MCHAR */
-  bool end_adjusted;            /* backuped b_op_end one char (only used by
-                                   do_format()) */
-  pos_T start;                  /* start of the operator */
-  pos_T end;                    /* end of the operator */
-  pos_T cursor_start;           /* cursor position before motion for "gw" */
-
-  long line_count;              /* number of lines from op_start to op_end
-                                   (inclusive) */
-  bool empty;                   /* op_start and op_end the same (only used by
-                                   op_change()) */
-  bool is_VIsual;               /* operator on Visual area */
-  bool block_mode;              /* current operator is Visual block mode */
-  colnr_T start_vcol;           /* start col for block mode operator */
-  colnr_T end_vcol;             /* end col for block mode operator */
-  long prev_opcount;            // ca.opcount saved for K_EVENT
```

Comment:
Still, I'm not happy with this change in this PR, as `count`s affect much code outside of `ops.c`. (it is also not consistent, there is still count parameters as `long`) Reverting all the changed `count` types back to `long` creates just two warnings in `ops.c`, which should be easy to fix.